core/deployment/src/main/java/io/quarkus/artemis/core/deployment/DevServicesArtemisProcessor.java

```diff
@@ -262,7 +262,7 @@ private static RunningDevService startArtemis(
                         containerAddress.getId(),
                         null,
                         urlPropertyName,
-                        containerAddress.getUrl()))
+                        String.format("tcp://%s", containerAddress.getUrl())))
```

Comment:
Strange [URL](https://en.wikipedia.org/wiki/URL#Syntax) has to start with a scheme. This is bad naming from Quarkus I presume?

Reply:
Indead is this a misleading naming in Quarkus as the ContainerAddress class contains only 3 fields for Docker container id, hostname/IP and a port.

Reply:
Thanks for the fix and the issue.